docs/sdks/client-sdks/javascript/precomputed-assignments.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Precomputed Assignments
+sidebar_position: 100
+---
+
+## Overview
+
+Precomputed assignments is a feature that allows you to receive assignments for all flags for a given user.
+
+The computation happens with a remote call to an Eppo Edge Function which is globally distributed to be as close to the user as possible. 
+Availability is backed by Eppo's global CDN.
+
+The new SDK methods are available in Eppo's Javascript SDK version `3.8.0` and above.
```

Comment:
I am considering deprecating versions 3.8.0 and 3.8.1. Could we make this 3.8.2?